docs/01-app/01-getting-started/01-installation.mdx

```diff
@@ -79,7 +79,7 @@ Create an `app` folder, then add a `layout.tsx` and `page.tsx` file. These will
   height="363"
 />
 
-Create a [root layout](/docs/app/building-your-application/routing/layouts-and-templates#root-layout-required) inside `app/layout.tsx` with the required `<html>` and `<body>` tags:
+Create a [root layout](/docs/app/api-reference/file-conventions/layout#root-layouts) inside `app/layout.tsx` with the required `<html>` and `<body>` tags:
```

Comment:
Is this the final location of the root layout docs? What I could observe, it changed 4 times in 6 months 😄:

- July 18th: https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required
- August 20th: https://nextjs.org/docs/app/building-your-application/routing/layouts-and-templates#root-layout-required
- October 22th: https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required
- November 10th: https://nextjs.org/docs/app/building-your-application/routing/layouts-and-templates#root-layout-required
- December 27th: https://nextjs.org/docs/app/api-reference/file-conventions/layout#root-layouts